### ISSUE TITLE
320x240@16c graphics driver

### ISSUE THREAD

**Issue by mist64** [commanderx16/x16-rom#99]
There is currently a 320x200@256c graphics driver in kernal/drivers/bitmap.s. We should also have a 320x240@16c driver that can be enabled with a different argument to SCREEN. The architecture already allows swapping the graphics driver at runtime, and high-level graphics functions like line drawing and fonts should just work with any driver.

**Comment by Kroc** [comment]
Could constants for bounds checks, such as 320 for the width, be moved to temporary zero-page locations? Not as fast as a hard-coded value in ROM, but potentially worth the space-savings from duplicating drivers.

**Comment by mist64** [comment]
Yes, I'm all for reusing code for similar modes. 640x400@16c and 320x200@16c can reuse the same driver, but 16c modes need a completely different driver than 256c modes.
